src/test/kotlin/ar/edu/unahur/obj2/semillasAlViento/ParcelaTest.kt

```diff
@@ -0,0 +1,64 @@
+package ar.edu.unahur.obj2.semillasAlViento
+
+import io.kotest.core.spec.style.DescribeSpec
+import io.kotest.matchers.shouldBe
+import io.kotest.assertions.throwables.shouldThrowAny
+import io.kotest.matchers.collections.shouldContain
+
+internal class ParcelaTest: DescribeSpec({
+    val parcela = Parcela(20, 1, 8)
+    describe ("Una parcela de 20m de ancho x 1m de largo") {
+
+        it("tiene una superficie de 20 m2") {
+            parcela.superficie().shouldBe(20)
+        }
+
+        it("soporta hasta 4 plantas") {
+            parcela.cantidadMaximaPlantas().shouldBe(4)
+        }
+
+        describe("se planta soja") {
+            for (num in 1..4) {
+                parcela.plantar(Soja(2019, 1.1f))
+            }
+
+            it("no tiene complicaciones si plantamos 4 plantas de soja") {
+                parcela.tieneComplicaciones().shouldBe(false)
+            }
+
+            it("error al agregar una quinta planta") {
+                shouldThrowAny {
+                    parcela.plantar(Soja(2019, 1.1f))
+                }
+            }
+        }
+    }
+
+    describe ("Una agricultora") {
+        val parcelaGrandeDeSoja = Parcela(40,4,10)
+        val parcelaDeSoja = Parcela(30,1,8)
+        val parcelaDeMenta = Parcela(5,4,3)
+        parcela.plantar(Soja(2006, 1.1f))
+        parcelaDeSoja.plantar(Soja(2009, 1.3f))
+        parcelaGrandeDeSoja.plantar(Soja(2019, 1.8f))
+        parcelaDeMenta.plantar(Menta(2020, 0.5f))
+        parcelaDeMenta.plantar(Menta(2018, 0.3f))
+        val agricultora = Agricultora(mutableListOf<Parcela>(parcela,
+                parcela,
+                parcelaDeSoja,
+                parcelaGrandeDeSoja,
+                parcelaDeMenta)
+        )
+        it("tiene parselas que dan semillas") {
+            agricultora.parcelasSemilleras().shouldContain(parcelaDeSoja)
+            agricultora.parcelasSemilleras().shouldContain(parcelaGrandeDeSoja)
+            agricultora.parcelasSemilleras().size.shouldBe(2)
```

Comment:
Podría resumirse así:
```suggestion
            agricultora.parcelasSemilleras().shouldContainExactly(parcelaDeSoja, parcelaGrandeDeSoja)
```